### DISCUSSION TITLE
Where are the profile infos in one of the js files?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [copy/v86#1161] [Q&A]
I have tried searching the repository for a section of a specific file that I can't remember where or which one it is, that has an object listing exact, speccific details about each demo(eg. where is the image, size, memory of it, state image, HDD image, etc.).
Does anyone know where this is?

Also sorry if this seems random and unnecessary, but I would like to examine how the demo profiles worked.

**Comment by FunnyCorgi** [comment]
I think this might be it?
https://copy.sh/v86/build/v86_all.js

**Comment by SuperMaxusa** [comment]
https://github.com/copy/v86/blob/f92e6b4b55c3f6e5a327b836375df20fedd45fa4/src/browser/main.js#L142-L153

**Reply by FunnyCorgi** [reply]
Thank you!
